src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -547,7 +565,20 @@ private JCheckBox getScaleImages() {
 		}
 		return scaleImages;
 	}
-	
+		
+	private JComboBox<String> getlookAndFeelSelect() {
+		if (lookAndFeel == null) {
+			lookAndFeel = new JComboBox<String>();
+			lookAndFeel.setMaximumRowCount(5);
+			UIManager.LookAndFeelInfo[] looks = UIManager.getInstalledLookAndFeels();
+			lookAndFeel.addItem(" ");	// Default look 

```

Comment:
It could be an empty string (like when read from the configurations).

Reply:
@thc202  Can you please elaborate on this? I didn't get your point.

Reply:
I was suggesting to use `""` instead of `" "`, it does not need the space.